### ISSUE TITLE
Multiple number formats for Indonesia (+62)

### ISSUE THREAD

**Issue by FreedomLabsIO** [MuhammedAlmaz/React-Native-INTL-Phone-Input#43]
It seems there are 2 number formats in Indonesia (+62)

+62 XXX XXXX XXX
+62 XXX XXXX XXXX

But on Countries.js the mask is fixed as
mask: '9999-999-999' so only the first one works.

Is it possible to add multiple masks ?

**Comment by SirPhemmiey** [comment]
Hey @brunoaduarte, did you find a fix for this?

**Comment by FreedomLabsIO** [comment]
> Hey @brunoaduarte, did you find a fix for this?

No... i've changed Countries.js to +62 XXX XXXX XXXXX and disabled number verification.
That's the only way out.

+62 XXX XXXX XXXXX
One more

So there are 3 formats
+62 XXX XXXX XXX
+62 XXX XXXX XXXX
+62 XXX XXXX XXXXX
